pkg/apis/nmstate/v1/unit_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package v1
```

Comment:
I think it should be named v1_suite_test.go

Reply:
Done. Also I don't know if we want different suite for policy in the future for example, but let's just remove and see what we need later on.